packages/react-charts/src/components/ChartUtils/chart-tooltip.ts

```diff
@@ -157,7 +157,7 @@ export const getLegendTooltipSize = ({
   // Get spacing to help align legend labels and text values
   const spacer = 'x';
   const getSpacing = (legendLabel: string, textLabel: string) => {
-    let spacing = '';
+    let spacing = '\u00A0';
```

Comment:
just a note that this is a non-breaking space

Reply:
The tooltip is also SVG, so there is no wrapping here